### PR TITLE
feat: Update Services Page

### PR DESCRIPTION
**Pull request by rut1303**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,13 +64,13 @@
   },
   "devDependencies": {
     "@types/styled-components": "^5.1.26",
+    "eslint": "^8.36.0",
```
Comment (vaidehi27) [inline package.json]: Is it needed?

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,52 @@
 import React from "react";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-
+import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
 import NavigationBar from "./Components/Navbar";
-import MyCarousel from "./Pages/Home";
-import AboutUs from "./Pages/About";
-import Home from "./Pages/Home";
-import DonationPage from "./Pages/Donation";
+import { LifeTimeMembership } from "./Pages/Servicespage/LifeTimeMembership";
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
import { LifeTimeMembership } from "./Pages/Services/LifeTimeMembership";
```

File: src/App.tsx [hunk 3]
```diff
@@ -1,24 +1,52 @@
...
     <>
       <Router>
         <NavigationBar />
         <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/Pages/Home" element={<MyCarousel />} />
-          <Route path="/Pages/About" element={<AboutUs />} />
-          <Route path="/pages/Donation" element={<DonationPage />} />
+          <Route
+            path="/services-page/life-time-membership"
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
            path="/services/life-time-membership"
```

File: src/App.tsx (hunk 2 above)
Comment (rut1303) [reply src/App.tsx]: Solved.

File: src/App.tsx (hunk 3 above)
Comment (rut1303) [reply src/App.tsx]: Solved.

File: package.json (hunk 1 above)
Comment (rut1303) [reply package.json]: Solved.

File: src/App.tsx [hunk 4]
```diff
@@ -1,18 +1,45 @@
 import React from "react";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import NavigationBar from "./Components/Navbar";
-import Home from "./Pages/Home";
-import FestivalPage from "./Pages/Festival";
+import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
+import { NavigationBar } from "./Components/Navbar";
 import { Contact } from "./Pages/Contact";
+import { LifeTimeMembership } from "./Pages/Services/LifeTimeMembership";
+import { IskconYouthForun } from "./Pages/Services/IskconYouthForum";
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
import { IskconYouthForum } from "./Pages/Services/IskconYouthForum";
```

File: src/Pages/Services/BookDistribution/constant.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import bookDistribution2 from "../../../assets/images/book-distribution2.jpg";
+import bookDistribution3 from "../../../assets/images/book-distribution3.jpg";
+import bookDistribution4 from "../../../assets/images/book-distribution4.jpg";
+import bookDistribution5 from "../../../assets/images/book-distribution5.jpg";
+import bookDistribution6 from "../../../assets/images/book-distribution6.jpg";
+import bookDistribution7 from "../../../assets/images/book-distribution7.jpg";
+import bookDistribution8 from "../../../assets/images/book-distribution8.jpg";
+import bookDistribution9 from "../../../assets/images/book-distribution9.jpg";
+
+export const bookDistributionImage = [
```
Comment (vaidehi27) [inline src/Pages/Services/BookDistribution/constant.ts]:
```suggestion
export const bookDistributionImages = [
```

File: src/Pages/Services/DeityWorship/constant.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import deityWorship1 from "../../../assets/images/deity-worship1.jpg";
+import deityWorship2 from "../../../assets/images/deity-worship2.jpg";
+import deityWorship3 from "../../../assets/images/deity-worship3.jpg";
+import deityWorship4 from "../../../assets/images/deity-worship4.jpg";
+import deityWorship5 from "../../../assets/images/deity-worship5.jpg";
+
+export const deityWorshipImage = [
```
Comment (vaidehi27) [inline src/Pages/Services/DeityWorship/constant.ts]:
```suggestion
export const deityWorshipImages = [
```

File: src/Pages/Services/PageLayout/Carousel.tsx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      id={id}
+      className="carousel slide"
+      data-bs-ride="carousel"
+      data-bs-interval="2000"
+    >
+      <div className="carousel-inner">
+        {images.map((image) => (
+          <div
+            key={Math.random()}
+            className={`carousel-item ${image === images[0] ? "active" : ""}`}
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/Carousel.tsx]:
```suggestion
        {images.map((image, index) => (
          <div
            key={Math.random()}
            className={`carousel-item ${index === 1 ? "active" : ""}`}
```

File: src/Pages/Services/PageLayout/style.ts [hunk 8]
```diff
@@ -0,0 +1,55 @@
+
+import styled from "styled-components";
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/style.ts]:
```suggestion
import styled from "styled-components";
```

File: src/Pages/Services/PageLayout/index.tsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+import titlebBaseImage from "../../../assets/images/title-base.png";
+import { Carousel } from "./Carousel";
+
+interface ServiceDiscriptionProps {
+  title: string;
+  content: string[];
+  imageUrl: string[];
+}
+
+export const ServiceDiscription: React.FC<ServiceDiscriptionProps> = ({
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/index.tsx]:
```suggestion
export const ServiceDiscription: React.FC<ServiceDescriptionProps> = ({
```

File: src/App.tsx (hunk 4 above)
Comment (Sanskarpanchal27) [reply src/App.tsx]: solved

File: src/Pages/Services/PageLayout/index.tsx (hunk 9 above)
Comment (Sanskarpanchal27) [reply src/Pages/Services/PageLayout/index.tsx]: solved

File: src/Pages/Services/PageLayout/style.ts (hunk 8 above)
Comment (Sanskarpanchal27) [reply src/Pages/Services/PageLayout/style.ts]: solved

File: src/Pages/Services/PageLayout/Carousel.tsx (hunk 7 above)
Comment (Sanskarpanchal27) [reply src/Pages/Services/PageLayout/Carousel.tsx]: solved

File: src/Pages/Services/DeityWorship/constant.ts (hunk 6 above)
Comment (Sanskarpanchal27) [reply src/Pages/Services/DeityWorship/constant.ts]: solved

File: src/Pages/Services/BookDistribution/constant.ts (hunk 5 above)
Comment (Sanskarpanchal27) [reply src/Pages/Services/BookDistribution/constant.ts]: solved

File: src/Pages/Services/PageLayout/index.tsx [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+        <Heading>{title}</Heading>
+        <ImageWrapper>
+          <TitleImage src={titlebBaseImage} alt="TitleBase" />
+        </ImageWrapper>
+        <Carousel images={imageUrl} />
+        <Paragraph>{mappedContent}</Paragraph>
+      </TextWrapper>
+    </div>
+  );
+};
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/index.tsx]:
```suggestion
  return (
    <div>
      <TextWrapper>
        <Heading>{title}</Heading>
        <ImageWrapper>
          <TitleImage src={titlebBaseImage} alt="TitleBase" />
        </ImageWrapper>
        <Carousel images={imageUrl} />
        <Paragraph>{content.map(paragraph => <p>{paragraph}</p>)}</Paragraph>
      </TextWrapper>
    </div>
  );
};
```

Comment (Priyansh1825) [reply src/Pages/Services/PageLayout/index.tsx]: resolved

File: src/Pages/Services/PageLayout/index.tsx [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+    <div>
+      <TextWrapper>
+        <Heading>{title}</Heading>
+        <ImageWrapper>
+          <TitleImage src={titlebBaseImage} alt="TitleBase" />
+        </ImageWrapper>
+        <Carousel images={imageUrl} />
+        <Paragraph>
+          {content.map((paragraph) => (
+            <p>{paragraph}</p>
```
Comment (vaidehi27) [inline src/Pages/Services/PageLayout/index.tsx]:
```suggestion
            <p key={paragraph}>{paragraph}</p>
```

Comment (Priyansh1825) [reply src/Pages/Services/PageLayout/index.tsx]: resolved

File: src/App.tsx [hunk 12]
```diff
@@ -3,10 +3,18 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { NavigationBar } from "./Components/Navbar";
 import { HomePage } from "./Pages/Home";
 import { DonationPage } from "./Pages/Donation";
-import { FestivalPage } from "./Pages/Festival";
 import { Contact } from "./Pages/Contact";
-import { VisitUs } from "./Pages/VisitUs";
+import { VisitUs } from "./Pages/Visitus";
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
import { VisitUs } from "./Pages/VisitUs";
```
Change file name if required

Comment (Priyansh1825) [reply src/App.tsx]: resolved

File: src/App.tsx [hunk 13]
```diff
@@ -4,7 +4,7 @@ import { NavigationBar } from "./Components/Navbar";
 import { HomePage } from "./Pages/Home";
 import { DonationPage } from "./Pages/Donation";
 import { Contact } from "./Pages/Contact";
-import { VisitUs } from "./Pages/Visitus";
+import { VisitUs } from "./Pages/VisitUs";
```
Comment (vaidehi27) [inline src/App.tsx:8]: package name is still `Visitus` and not `VisitUs`

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**